packages/utils/telemetry-utils/src/test/utils.spec.ts

```diff
@@ -30,3 +31,15 @@ describe("logIfFalse", () => {
 		assert.strictEqual(logger.events.length, 0);
 	});
 });
+
+describe("tagData", () => {
+	it("tagData", () => {
+		const taggedData = tagData(TelemetryDataTag.CodeArtifact, { foo: "bar" });
```

Comment:
Probably should add an `undefined` case here given that's where all the complexity is in the implementation (and typings!)